code/Kevin/Django/lab3/texas_project/templates/home.html

```diff
@@ -0,0 +1,51 @@
+{% extends 'base.html' %}
+
+
+{% block content %}
+<div>
+    <a href="{% url 'login' %}">Login</a>
+    <a href="{% url 'users:signup' %}">Sign up</a>
+    {% if user.is_authenticated %}
+      <a href="{% url 'logout' %}">Logout</a>
+    {% endif %}
+</div>
+<div class="border-2 h-32 flex justify-center items-center">
+  {% if user.is_authenticated %}
+    <a  href="{% url 'texas_app:create' %}"><button class="bg-blue-500 hover:bg-blue-700 text-white font-bold rounded p-2">Contribute!</button></a>
+  {% endif %}
+  {% if user.is_authenticated == False%}
+    <a  href="{% url 'login' %}"><button class="bg-blue-500 hover:bg-blue-700 text-white font-bold rounded p-2">Contribute!</button></a>
+  {% endif %}
+  </div>
+
+  <div class="flex justify-center">
+    {% for post in object_list %}
```

Comment:
Not entirely sure what is causing it to happen but something is causing your `post.body` to spill out of its box. Im not too familiar with Tailwind. I imagine recreating the div from scratch you might be able to find  where the error is. Otherwise, your models and routes all work. Great work.

Reply:
<img width="944" alt="lonestar" src="https://user-images.githubusercontent.com/100663399/198156342-be7bea96-1ae1-4196-a034-b2648cc5c8f0.png">